worldengine/draw.py

```diff
@@ -403,6 +403,8 @@ def draw_satellite(world, target):
     # Get an elevation mask where heights are normalized between 0 and 255
     elevation_mask = get_normalized_elevation_array(world)
 
+    rng = numpy.random.RandomState(world.seed)  # create our own random generator; necessary for now to make the tests reproducible, even though it is a bit ugly
```

Comment:
Since the test-suite can call this function, I had to do something about the RNG to give the test a chance to pass. The remaining problem doesn't seem to be random.